hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/client/functional/TestHoodieBackedMetadata.java

```diff
@@ -556,23 +554,10 @@ public void testUpdationOfPopulateMetaFieldsForMetadataTable() throws Exception
     tableType = COPY_ON_WRITE;
     init(tableType, false);
 
+    // populateMeta fields is false by default
     writeConfig = getWriteConfigBuilder(true, true, false)
         .withMetadataConfig(HoodieMetadataConfig.newBuilder()
             .enable(true)
-            .withPopulateMetaFields(true)
-            .build())
-        .build();
-    initWriteConfigAndMetatableWriter(writeConfig, true);
-    doWriteOperation(testTable, "0000001", INSERT);
```

Comment:
We still need this to write the first batch?  If not, change the latter commit time `"0000002"` to `"0000001"`?